app/views/users/passwords/edit.html.erb

```diff
@@ -1,26 +1,36 @@
 <% self.title = t('titles.edit_info.password') %>
 
+<% if @update_password_presenter.required_password_change? %>
+  <%= render AlertComponent.new(
+        type: :warning,
+        class: 'margin-bottom-4',
+      ) do
+        t('users.password_compromised.warning', app_name: APP_NAME)
+      end
+  %>
+<% end %>
+
 <%= render PageHeadingComponent.new.with_content(t('headings.edit_info.password')) %>
 
-<p id="password-description">
```

Comment:
I'm actually kinda curious why we don't show this text when the user has to change it. It has important information like length requirements and forbidden passwords including common phrases and repeated characters. Maybe it was an oversight from the design? It'd make it a lot simpler implementation too if we kept it consistent.

Reply:
Yea I agree, Idk im fine with keeping it.

Reply:
Can you run this past UX folks on the team?

Reply:
@mdiarra3 Were you able to follow-up on this thread discussion?